src/app/(sidebar)/xdr/to/page.tsx

```diff
@@ -136,7 +136,12 @@ export default function ToXdr() {
           <SdsLink href="https://developers.stellar.org/docs/learn/encyclopedia/data-format/xdr-json">
             XDR ⇄ JSON guide
           </SdsLink>{" "}
-          on the Stellar Developer Docs.
+          on the Stellar Developer Docs. To see the XDR ⇄ JSON conversion
+          specification, please see{" "}
+          <SdsLink href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0051.md">
```

Comment:
Fyi protocol docs are also available on these swish URLs:
```suggestion
          <SdsLink href="https://stellar.org/protocol/sep-51">
```